source/multigrid/mg_tools.cc

```diff
@@ -332,7 +326,7 @@ namespace MGTools
     // unknown number of cells may
     // share an object of dimension
     // smaller than dim-1.
-    for (cell = dofs.begin_active(); cell != end; ++cell)
+    for (const auto &cell : dofs.active_cell_iterators_on_level(level))
```

Comment:
The original code is confusing, the for loop starts at `dofs.begin_active()` but ends at `dofs.end(level)`.

Reply:
I am not sure that the old code is correct. I would go from begin(level) to end(level).

Reply:
Hmm... I am not sure either, but for me it sounds more reasonable to loop over active cells on each level if we want to find out about the number of coupling degrees of freedom on a particular level. Unfortunately, there is no test for this function.

Reply:
We are asking for ``get_mg_dof_indices``, so I don't see a reason for looking at active cells only.

Reply:
Thinking some more about it, I guess you're right. I changed the pull request accordingly.